pkg/openstack/loadbalancer.go

```diff
@@ -2283,6 +2316,10 @@ func (lbaas *LbaasV2) ensureAndUpdateOctaviaSecurityGroup(clusterName string, ap
 			return fmt.Errorf("failed to create Security Group for loadbalancer service %s/%s: %v", apiService.Namespace, apiService.Name, err)
 		}
 		lbSecGroupID = lbSecGroup.ID
+		if svcConf.supportLBTags {
+			lbSecGroup.Tags = lbaas.getCustomLoadBalancerTags(apiService)
+		}
```

Comment:
In this case you don't need to check `supportLBTags`, SGs are part of Neutron, not Octavia and we can assume they always support tagging. Also this is not the way to do it. In Neutron tagging is an operation separate from resource creation. I've provided an example of how to do it in this comment: https://github.com/kubernetes/cloud-provider-openstack/pull/2439#issuecomment-1785728438

Reply:
noted and fixed !

thank you for the clarification